src/Compiler/Compiler.cpp

```diff
@@ -1754,6 +1759,21 @@ bool Compiler::RegisterCommands(TclInterpreter* interp, bool batchMode) {
   };
   interp->registerCmd("wave_refresh", wave_refresh, this, nullptr);
 
+  auto diagnostic = [](void* clientData, Tcl_Interp* interp, int argc,
```

Comment:
That diagnostic command should only set the option  (like your packer "debug"):
compiler->PackOpt(Compiler::PackingOpt::Debug)

The usage would be 
...
diagnostic "packer" ... eventually more options here
packing
...

Reply:
this is not what we agreed with @ravic-rs

Reply:
I misinterpreted the requirement that diagnostic "packer" will run "packing" directly but not as a settings option before running the "packing"

Reply:
We will rework this to support diagnostics for other tasks, for now this works.